src/platform/site-wide/user-nav/components/SearchHelpSignIn.jsx

```diff
@@ -70,13 +69,17 @@ class SearchHelpSignIn extends React.Component {
     return (
       <div className="profile-nav">
         <SearchMenu
-          isOpen={this.props.isMenuOpen.search}
           clickHandler={this.handleSearchMenuClick}
+          isOpen={this.props.isMenuOpen.search}
         />
-        <HelpMenu
-          isOpen={this.props.isMenuOpen.help}
-          clickHandler={this.handleHelpMenuClick}
-        />
+        <a
+          className="vads-u-color--white vads-u-text-decoration--none vads-u-padding-top--0p5 vads-u-padding-x--1"
+          href="tel:+18006982411"
+          onClick={() => recordEvent({ event: 'nav-jumplink-click' })}
+          style={{ fontWeight: '600' }}
```

Comment:
`vads-u-font-weight--bold` is 700 unfortunately 😭

Reply:
@kelsonic is is possible to add a new class for this instead of using the inline CSS?
@cvalarida is this a style we needed added into the system or should we reach out to design to get it set to the 700 weight?

Reply:
600 is the font-weight that the rest of the navbar uses? Or part of the design spec? Also wondering if we can get away w/o inline CSS

Reply:
600 is both the font-weight that the rest of the footer uses + it's part of the design spec

Reply:
The existing SCSS can't be tweaked to include this link in its style rule for the bold? (Also we _are_ talking about the navbar here and not the footer right? Or am I confused?)

Reply:
Navbar I meant haha sorry 🤪 idk I usually don't add scss that often in favor of design utility class names

Reply:
Updated 👍